services/cd-service/pkg/repository/transformer.go

```diff
@@ -124,6 +124,23 @@ func UpdateDatadogMetrics(state *State) error {
 	return nil
 }
 
+func SendRegularlyDatadogMetrics(repo Repository, interval int, callBack func(repository Repository)) {
```

Comment:
```suggestion
func SendRegularlyDatadogMetrics(repo Repository, interval int, callBack func(repository Repository)) {
```
either make the `interval` a `Duration` type, or rename it to `intervalSeconds`.

Reply:
Fixed, changed interval to a Duration